aip/general/0158.md

```diff
@@ -92,6 +92,26 @@ message ListBooksResponse {
   - This total **may** be an estimate (but the API **should** explicitly
     document that).
 
+### Skipping results
+
+The request definition for a paginated operation **may** define an `int32 skip`
```

Comment:
Perhaps (or `int64 skip`) to match https://google.aip.dev/132

Or should we more generally recommend int64 here? One disadvantage of int64 I'm aware of is that endpoints generated JSON over HTTP changes the type to string (rather than integer for int32).

Seems reasonable that some resources may grow to more than 2 billion.

Reply:
I expect if a collection grows to 10^9, jumping to the (10^9 + 1)th entry is unlikely to be an important use case. When is the last time you asked for page 200,000,001 in a Google search?